server/portal_server.js

```diff
@@ -113,8 +125,9 @@ app.post('/upload/:id', (req, res, next) => {
 
   req.pipe(req.busboy);
   req.busboy.on('file', (fieldname, file, filename) => {
-    console.log('Uploading: ' + filename);
-    const protocol = conf.env === 'development' ? req.protocol : 'https';
+    log.info('Uploading:', req.params.id);
+
+    const protocol = !notLocalHost ? req.protocol : 'https';
```

Comment:
how about `const protocol = notLocalHost ? 'https' : req.protocol;` to eliminate the double-negative?